lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -80,6 +80,7 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
   switch (NI.getKind()) {
   case Kinded::Kind::BatchedReduceMinNodeKind:
   case Kinded::Kind::BatchedReduceMaxNodeKind:
+  case Kinded::Kind::BatchedReduceProdNodeKind:
```

Comment:
Here you're claiming support for Float16 and BFloat16, but there is no test coverage in OperatorTest. Can you add a couple extra tests to cover this?

Reply:
Added test cases for Float16 and BFloat16 in OperatorTest.cpp